frontend/src/pages/DeviceSettingsPage/DeviceSettingsPage.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { ApplicationState, Dispatch } from '../../store'
+import { DeviceHeaderMenu } from '../../components/DeviceHeaderMenu'
+import { List, Typography } from '@material-ui/core'
+import { DeviceNameSetting } from '../../components/DeviceNameSetting'
+import { TestUI } from '../../components/TestUI'
+import { Gutters } from '../../components/Gutters'
+import { TagEditor } from '../../components/TagEditor'
+import { ListItemSetting } from '../../components/ListItemSetting'
+
+type Props = {
+  targetDevice: ITargetDevice
+  device: IDevice
+}
+
+enum notificationType {
+  'ONW' = 'onlineDeviceNotification',
+  'DEVICE_SHARED' = 'onlineSharedDeviceNotification',
+  'EMAIL' = 'notificationEmail',
+}
+
+const NOTIFICATION = {}
+export const DeviceSettingsPage: React.FC<Props> = ({ targetDevice, device }) => {
+  const { isOwner } = useSelector((state: ApplicationState) => ({
+    isOwner: device.owner.email === state.auth.user?.email,
+  }))
+
+  const { devices } = useDispatch<Dispatch>()
+  const [emailNotification, setEmailNotification] = useState(device.attributes.notificationEmail || false)
+  const [notificationInApp, setNotificationInApp] = useState(
+    isOwner ? device.attributes.onlineDeviceNotification : device.attributes.onlineSharedDeviceNotification || false
+  )
+
+  const emailNotifications = () => {
+    devices.setNotifications({
+      id: device.id,
+      name: notificationType.EMAIL,
+      value: !emailNotification,
+    })
+    setEmailNotification(!emailNotification)
+  }
+
+  const inAppNotifications = () => {
+    devices.setNotifications({
+      id: device.id,
+      name: isOwner ? notificationType.ONW : notificationType.DEVICE_SHARED,
+      value: !notificationInApp,
+    })
+    setNotificationInApp(!notificationInApp)
+  }
+
+  if (!device) return null
+
+  return (
+    <DeviceHeaderMenu device={device}>
+      <Typography variant="subtitle1">General</Typography>
+      <List>
+        <DeviceNameSetting device={device} targetDevice={targetDevice} />
+      </List>
+      <TestUI>
+        <Gutters inset>
+          <TagEditor device={device} />
+        </Gutters>
+      </TestUI>
+      <Typography variant="subtitle1">Device notifications</Typography>
+      <List>
+        <ListItemSetting
+          label="Email notifications"
+          icon="envelope"
+          toggle={emailNotification}
+          onClick={emailNotifications}
+          quote={false}
+        />
+        <ListItemSetting
+          label="In-app notifications"
+          icon="laptop"
+          toggle={notificationInApp}
```

Comment:
```suggestion
          toggle={!!notificationInApp}
```
So that the toggle appears by default